week0001/FizzBuzzMichal.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+for($counter = 1; $counter <= 100; $counter++)
+{
+    echo 0 == $counter%3 ? 'Fizz<br>' : '';
```

Comment:
You still invoke the function `echo` even when the condition is `false`. It's advised to avoid printing an empty string and save the call.